packages/core/src/browser/keybinding.ts

```diff
@@ -556,7 +554,7 @@ export class KeybindingRegistry {
             return false;
         }
 
-        for (const binding of bindings) {
+        for (const binding of [...bindings].reverse()) {
```

Comment:
spreading is unnecessary

Reply:
reverse() changes the bindings in place so without spreading the next time that piece of code is hit the binding orders are completely off